docs/UserGuide.md

```diff
@@ -169,7 +174,7 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [role/ROLE] [t/TAG]…​`
+Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [role/ROLE] [t/TAG] [r/REMARK] [dept/DEPARTMENT] [job/JOBTITLE] [skills/SKILLS] [prod/PRODUCTS] [tos/TERMSOFSERVICE] [pref/PREFERENCES]`
```

Comment:
I don't think we should show all these field for a person (any role)

Instead we may not include the fields specific to each role, and add three more line to include a role-specific prefixes

Reply:
one of the issue i receive ask me to specify if I don't understand wrongly https://github.com/AY2324S2-CS2103T-F12-1/tp/issues/174